### DISCUSSION TITLE
FR: Enhance FilePicker UX for directory mode

### DISCUSSION THREAD

**Discussion by thiemotorres** [charmbracelet/huh#558] [Ideas]
**Is your feature request related to a problem? Please describe.**
When using the filepicker to pick directories, selecting and opening directories uses the same key (enter). Also when just wanting to select directories, being able to see all files is distracting.

**Describe the solution you'd like**
When picking directories I would like to have a different default keymapping (e.g. space to open directories and enter to select). An option to just show directories and hide all files would be awesome

**Comment by bashbunni** [comment]
Hey @thiemotorres, thank you so much for wanting to make this project better! I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.

If you (or anyone else interested in this feature) have a demonstration of the current behaviour, please include it in this conversation too. It will allow us to address this request more quickly 🙏
